### PR TITLE
[MOB-17462] Increase test coverage

### PR DESCRIPTION
**Pull request by emdobrin**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: code/edgebridge/src/test/java/com/adobe/marketing/mobile/edge/bridge/ListenerGenericTrackRequestContentTest.java [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+		// test
+		listener.hear(event);
+
+		// verify
+		testExecutor.awaitTermination(100, TimeUnit.MILLISECONDS);
+		verify(mockExtension, times(0)).handleTrackRequest(any(Event.class));
+	}
+
+	@Test
+	public void testHear_WhenEventNull() throws InterruptedException {
```
Comment (kevinlind) [inline code/edgebridge/src/test/java/com/adobe/marketing/mobile/edge/bridge/ListenerGenericTrackRequestContentTest.java:80]: Can you add a test with null EventData? That should get this class coverage to 100%.

File: code/edgebridge/src/test/java/com/adobe/marketing/mobile/edge/bridge/ListenerRulesEngineResponseContentTest.java [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+		// test
+		listener.hear(event);
+
+		// verify
+		testExecutor.awaitTermination(100, TimeUnit.MILLISECONDS);
+		verify(mockExtension, times(0)).handleRulesEngineResponse(any(Event.class));
+	}
+
+	@Test
+	public void testHear_WhenEventNull() throws InterruptedException {
```
Comment (kevinlind) [inline code/edgebridge/src/test/java/com/adobe/marketing/mobile/edge/bridge/ListenerRulesEngineResponseContentTest.java:80]: Can you add a test with null EventData? That should get this class coverage to 100%.

Comment (emdobrin) [reply code/edgebridge/src/test/java/com/adobe/marketing/mobile/edge/bridge/ListenerRulesEngineResponseContentTest.java:80]: Seems like the event data serializer sets data to empty map when null is provided, and even though we cannot test it I I would still keep it as a safeguard in case of core changes
